src/stylesheets/elements/form-controls/_checkbox-and-radio.scss

```diff
@@ -57,99 +233,18 @@
-  background-color: color("primary");
-  box-shadow: 0 0 0 units($theme-input-select-border-width) color("primary");
-}
-
-.usa-radio__input:checked + .usa-radio__label::before {
-  box-shadow: 0 0 0 units($theme-input-select-border-width) color("primary"),
-    inset 0 0 0 units($theme-input-select-border-width) color("white");
-
-  @media print {
-    box-shadow: inset 0 0 0 units($theme-input-select-border-width)
-        color("white"),
-      inset 0 0 0 units(2) color("primary"),
-      0 0 0 units($theme-input-select-border-width) color("primary");
-  }
-}
-
-.usa-checkbox__input:checked + .usa-checkbox__label::before,
-.usa-checkbox__input:checked:disabled + .usa-checkbox__label::before {
-  @include add-background-svg("correct8");
-  background-position: center center;
-  background-size: units(1.5) auto;
-
-  @media print {
-    background-image: none;
-    background-color: color("white");
-    content: url("#{$theme-image-path}/checkbox-check-print.svg");
-    text-indent: 0;
-  }
-}
-
-.usa-radio__input:focus + .usa-radio__label::before {
-  @include focus-outline(null, null, null, 0.5);
-}
-
-.usa-checkbox__input:disabled + .usa-checkbox__label,
-.usa-radio__input:disabled + .usa-radio__label {
-  color: color("disabled");
-  cursor: not-allowed;
-}
-
-.usa-checkbox__input:focus + .usa-checkbox__label::before {
-  @include focus-outline;
-}
-
-.usa-checkbox__input:disabled + .usa-checkbox__label::before,
-.usa-radio__input:disabled + .usa-radio__label::before {
-  background: color("disabled-light");
-  box-shadow: 0 0 0 units($theme-input-select-border-width) color("disabled");
-  cursor: not-allowed;
-}
-
-.usa-radio__input:checked:disabled + .usa-radio__label::before {
-  background: color("disabled-dark");
-  box-shadow: 0 0 0 2px color("disabled-dark"),
-    inset 0 0 0 2px color("disabled-light");
-}
-
-// Checkboxes and radios with tap-friendly targets
-.usa-checkbox__input--tile + .usa-checkbox__label,
-.usa-radio__input--tile + .usa-radio__label {
-  border: units($theme-input-tile-border-width) solid
-    color($theme-input-tile-border-color);
-  border-radius: radius($theme-input-tile-border-radius);
-  margin-top: units(1);
-  padding: units(1.5) units(2) units(1.5) units(5);
-}
-
-.usa-checkbox__input--tile:checked + .usa-checkbox__label,
-.usa-radio__input--tile:checked + .usa-radio__label {
-  background-color: color($theme-input-tile-background-color-selected);
-  border-color: color($theme-input-tile-border-color-selected);
-}
-
-.usa-checkbox__input--tile:disabled:checked + .usa-checkbox__label,
-.usa-radio__input--tile:disabled:checked + .usa-radio__label {
-  background-color: color("disabled-light");
-  border-color: color($theme-input-tile-border-color);
-}
-
 .usa-checkbox__label-description,
 .usa-radio__label-description {
   display: block;
   font-size: size("ui", "2xs");
   margin-top: units(1);
   text-indent: 0;
 }
+
+// Test code for scoped custom colors
```

Comment:
This test code is used to power the example here: https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/jm-tile-add-bg/components/preview/checkbox--test.html

We should comment it out before going into production